openlibrary/api.py

```diff
@@ -112,10 +112,10 @@ def get(self, key, v=None):
         response = self._request(key + '.json', params={'v': v} if v else {})
         return unmarshal(response.json())
 
-    def get_many(self, keys, max_length = 500):
+    def get_many(self, keys):
         """Get multiple documents in a single request as a dictionary.
         """
```

Comment:
Some callers might only want the first 10 results.
```suggestion
    def get_many(self, keys, max_length = 100):
        """Get multiple documents in a single request as a dictionary.
        """
        max_length = min(max_length, 100)
```

Reply:
This function doesn't appear to have any callers outside of copydocs 🙃